src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Colors[] COLORS = Colors.values();
+    private static final int COLORS_COUNT = COLORS.length;
+    private Random random = new Random();
+
+    private enum Colors {
+        BLACK,
+        RED,
+        WHITE,
+        YELLOW
+    }
```

Comment:
```suggestion
    private enum Color {
        BLACK,
        RED,
        WHITE,
        YELLOW
    }
```
This enum should not be in the CollorSupplier, create it separately